docs/developers/SSV-SDK/examples/bulk-register-validators.md

```diff
@@ -12,15 +12,27 @@ This page show show to register any amount of validators to the SSV network.&#x2
 Prerequisite: This tutorial assumes you already have keystores generated, or will use the [code illustrated here](create-validator-keys.md) to generate them pragmatically.
 :::
 
-#### 1. Import <a href="#id-1-installation" id="id-1-installation"></a>
+
+Bulk Register flow:
+
+![Get Started](/img/get-started-2.avif)
+
+Running a distributed validator is a process composed of these steps (outlined in the above schema):
+
+1. Select the cluster of operators to manage your validators.
```

Comment:
It does not look like steps 1, 2, 3, 4 are referenced throughout the example. They should be mentioned when the piece of code actually accomplishes the step itself.